src/System.Device.Ports/System.Device.Ports.SerialPort/System.Device.Ports.SerialPort.csproj

```diff
@@ -0,0 +1,57 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net6.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(NETCoreSdkRuntimeIdentifier.StartsWith('win'))">$(TargetFrameworks);net6.0-windows10.0.17763.0</TargetFrameworks>
+    <LangVersion>9</LangVersion>
+    <Nullable>enable</Nullable>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <IsPackable>true</IsPackable>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Remove="winmd\**" />
+    <EmbeddedResource Remove="winmd\**" />
+    <None Remove="winmd\**" />
+    <Compile Remove="Interop.Unix.DefaultPathBufferSize.cs" />
+    <None Include="buildTransitive\net5.0\System.Device.Ports.targets" Pack="true" PackagePath="\buildTransitive\net5.0" />
+    <None Include="Interop\Unix\Interop.Unix.DefaultPathBufferSize.cs" />
+    <PackageReference Include="Microsoft.DotNet.GenAPI" Version="$(MicrosoftDotNetGenApiPackageVersion)">
+      <PrivateAssets>all</PrivateAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <PackageReference Include="System.Memory" Version="$(SystemMemoryPackageVersion)" />
+    <PackageReference Include="System.Threading.Tasks.Extensions" Version="$(SystemThreadingTasksExtensionsPackageVersion)" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
+    <!--Remove the Linux-specific code-->
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+    <!--Remove Windows Code-->
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="Resources\Strings.Designer.cs">
```

Comment:
Why do we need a Strings.Designer? I would imagine you should just need the resx file bellow (without metadata) for all of our strings. My advise here is to do what a traditional project would do, so perhaps even re-add the strings.resx from Visual Studio which should add the right modifications into this .csproj.

Reply:
I just added a resx to the project. It probably used the VS template from Winform or similar.
I now copied the file, removed it from VS and re-added. Let's see if that's better.